etna/transforms/missing_values/resample.py

```diff
@@ -44,13 +44,19 @@ def _get_folds(self, df: pd.DataFrame) -> List[int]:
         Here the ``in_column`` frequency gap is divided into the folds with the size of dataset frequency gap.
         """
         in_column_index = df[self.in_column].dropna().index
-        if len(in_column_index) <= 1 or (len(in_column_index) >= 3 and not pd.infer_freq(in_column_index)):
+
+        fail_datetime_timestamp = (
+            in_column_index.dtype != "int" and len(in_column_index) >= 3 and not pd.infer_freq(in_column_index)
+        )
+        if len(in_column_index) <= 1 or fail_datetime_timestamp:
             raise ValueError(
-                "Can not infer in_column frequency!"
+                "Can not infer in_column frequency! "
```

Comment:
?

Reply:
I think we should add a space. Otherwise messages collapse together and it doesn't look nice.